onnxruntime/core/providers/dml/DmlExecutionProvider/src/MLOperatorAuthorImpl.h

```diff
@@ -309,6 +309,7 @@ class OnnxTensorWrapper : public WRL::Base<IMLOperatorTensor>, public Closable
  private:
     size_t m_tensorByteSize = 0;
     std::unique_ptr<std::byte[]> m_unpackedTensor;
```

Comment:
You think it makes sense to (rather than have two separate byte blobs) have `MLOperatorAuthorImpl.cpp UnpackTensor` just return a vector, and use it for both branches? It appears it's not called in many places, mainly `OnnxTensorWrapper::OnnxTensorWrapper`. I'm fine either way.

Reply:
I think this will change to a pointer anyway when merging with this PR to avoid the extra zeroing.

[https://github.com/microsoft/onnxruntime/pull/19220](https://github.com/microsoft/onnxruntime/pull/19220)